src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    private static final DateConverter DATE_CONVERTOR = new DateConverter();
```

Comment:
```suggestion
    private final DateConverter dateConverter = new DateConverter();
```
if our service is dependent on another service - the dependency should not be a constant